source/process/documentation-UItext-guidelines.rst

```diff
@@ -0,0 +1,193 @@
+
+- Use complete sentences.
+- Include a question that has a Yes/No answer.
+- Examples:
+
+  - *Are you sure you want to delete this channel?*
+  - *A plug-in with this ID already exists. Would you like to overwrite it?*
+
+Warning messages
+~~~~~~~~~~~~~~~~
+
+A warning message is for alerting somebody about something that might go wrong. The user can continue using Mattermost unless the warning message needs an explicit user input.
+
+- Use complete sentences.
+- Explain what has happened or can happen, and what can go wrong as a consequence.
+- If the message contains a question, phrase it in such a manner so that it has a Yes/No answer (unless you have specific action buttons for the message).
+
+- Examples:
+
+  - *The Enterprise license expires in 2 days. If you do not renew it, some features will be disabled on licence expiry.*
+  - *If you claim this AD/LDAP account, you will no longer be able to log in with your email. Do you want to continue?*
+
+Error messages
+~~~~~~~~~~~~~~~
+
+An error message is for telling somebody that something went wrong. Errors prevent a user from doing a task or accessing a feature till the error is resolved.
+
+- Use complete sentences.
+- If what went wrong isn't obvious, explain in one sentence.
+- If a solution or workaround isn't obvious, suggest one.
+- Examples:
+
+  - *This message is too long. Shorten it to 120 characters.*
+  - *The passwords do not match.*
+
+
+Guidelines for UI elements
+---------------------------
+Use this table when writing the text for UI elements such as windows, dialog boxes, labels, and prompts.
+
+.. list-table::
+   :widths: 10 30 10 15 35
+   :header-rows: 1
+
+   * - Element
+     - Image 
+     - Capitalization
+     - Phrasing
+     - Examples     
+   * - Menu
+     - |menu.png|
+     - All words
+     - - Noun, noun phrase, or verb
+       - No punctuation
+       - Not more than three words
+     - - *Members*
+       - *Account Preferences*
+       - *Log Out*
+   * - Tooltip
+     - |tooltip.png|
+     - First word
+     - - Sentence fragment or sentence
+       - No punctuation
+       - Include articles (*a*, *an*, *the*)
+     - - *Start a Zoom meeting*
+       - *Flag for following up*
+       - *Remove from this list*
+   * - In-field text
+     - |field.png|  
+     - First word
+     - - Sentence fragment, sentence, or word
+       - No punctuation
+       - Include articles (*a*, *an*, *the*)
+     - - *Add a comment*
+       - *Search*
+   * - Action button
+     - |action.png|
+     - All words
+     - - Verb or verb phrase
+       - No articles (*a*, *an*, *the*)
+       - Exceptions: *OK*, *Yes*, *No*
+     - - *Add Comment*
+       - *Edit*
+   * - Label before a UI element
+     - |label_before.png|
+     - First word
+     - - Noun, verb, or sentence fragment
+       - End with a colon
```

Comment:
Is a colon applicable in all of the cases of this category you found in the spreadsheet?

Reply:
No, a colon is applicable only if it precedes another UI element such as a radio button or a check box.

Reply:
Okay, can we make sure the guidelines state that specifically. I'd like to try and get the guidelines to cover as many cases as possible so there is no (or as limited as possible) ambiguity for someone reading these for the first time with no context. That applies to all sections if there are cases we should call out specifically

Reply:
Added "End with a colon if it precedes another UI element such as a radio button or check box"